.circleci/config.yml

```diff
@@ -692,7 +692,7 @@ jobs:
     test--test_postake_txns_medium_curves:
         resource_class: large
         docker:
-            - image: codaprotocol/coda:toolchain-6b64dedbb6e27f0cc57228f1b44aaa87da489e83
+            - image: codaprotocol/coda:toolchain-bac894a089c02852067dac428c0c6a57ff15b4f3
```

Comment:
I wonder if we can alias these docker images with a version number rather than a hash?

Reply:
If so, that's outside the scope of this PR.

I'm not sure versioning the Docker images is a good idea. The hashes come from a particular commit, making it easy to tie the images to the git commit history. Using the hashes is also convenient for automating the image builds.

Reply:
Yes, I agree the git sha is nice (but also I was the one that decided it originally, so open to disagreement)